CHANGELOG.md

```diff
@@ -47,6 +47,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fix stale timestamps reported by the lastvalue aggregation. (#5517)
 - Indicate the `Exporter` in `go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploghttp` must be created by the `New` method. (#5521)
 
+<!-- Released section -->
+<!-- Don't change this section unless doing release -->
```

Comment:
Shouldn't moving this be documented in the release issue template?

Reply:
I added this in the `RELEASING.md` file.